hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestNodeDecommissionManager.java

```diff
@@ -115,19 +114,19 @@ private ContainerInfo getMockContainer(ReplicationConfig rep, ContainerID conId)
   @Test
   public void testHostStringsParseCorrectly()
       throws InvalidHostStringException {
-    HostDefinition def = new HostDefinition("foobar");
+    NodeDecommissionManager.HostDefinition def = new NodeDecommissionManager.HostDefinition("foobar");
     assertEquals("foobar", def.getHostname());
     assertEquals(-1, def.getPort());
 
-    def = new HostDefinition(" foobar ");
+    def = new NodeDecommissionManager.HostDefinition(" foobar ");
     assertEquals("foobar", def.getHostname());
     assertEquals(-1, def.getPort());
 
-    def = new HostDefinition("foobar:1234");
+    def = new NodeDecommissionManager.HostDefinition("foobar:1234");
     assertEquals("foobar", def.getHostname());
     assertEquals(1234, def.getPort());
 
-    def = new HostDefinition("foobar.mycompany.com:1234");
+    def = new NodeDecommissionManager.HostDefinition("foobar.mycompany.com:1234");
```

Comment:
Please revert the changes in this file.  It doesn't change anything. We should avoid unnecessary code changes in commit.